homeassistant/components/proxmoxve/__init__.py

```diff
@@ -91,100 +94,132 @@
 
 async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
     """Set up the platform."""
-    hass.data.setdefault(DOMAIN, {})
+    # import to config flow
+    if DOMAIN in config:
+        for conf in config[DOMAIN]:
+            hass.async_create_task(
+                hass.config_entries.flow.async_init(
+                    DOMAIN, context={"source": SOURCE_IMPORT}, data=conf
+                )
+            )
```

Comment:
This would be a good place to add a non-fixable issue to the Repairs registry (to proactively notify the user that they should plan on removing the YAML from their `configuration.yaml`). Example: https://github.com/home-assistant/core/blob/b2f86ddf76afc37f578f174db9fca148fefed6c3/homeassistant/components/demo/__init__.py#L191-L200

Reply:
I chose to handle this in the import in the config flow… there I create the custom issue according to the import (successfully imported, import error, already exists, etc…).  Do you think a generic is needed here?

Reply:
My suggestion here creates an issue if the YAML still exists; if you handle the same use case in a more granular fashion downstream, that's fine.

Reply:
I created a specific one to warn about YAML deprecation.